lint/highlight.py

```diff
@@ -387,47 +430,82 @@ def draw(self, view):
         since each one potentially needs a different color.
 
         """
-        self.set_mark_style()
-
-        gutter_regions = {WARNING: [], ERROR: []}
-        draw_gutter_marks = persist.settings.get('gutter_theme') != 'None'
+        self.set_mark_style()  # TODO: does this still make sense?
```

Comment:
I think most of set_mark_style doesn't make sense anymore, although it also contains a check for show_marks_in_minimap. Needs a look anyway.

Reply:
`self.set_mark_style()` is still needed to set the visibility of marks in the minimap. 
I have adapted the code respectively.